src/bufr/BufrParser/Exports/Variables/TimeoffsetVariable.cpp

```diff
@@ -0,0 +1,168 @@
+/*
+ * (C) Copyright 2022 NOAA/NWS/NCEP/EMC
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+#include <iomanip>
+#include <iostream>
+#include <locale>
+#include <ostream>
+#include <ctime>
+#include <vector>
+#include <unordered_map>
+
+#include "eckit/exception/Exceptions.h"
+#include "oops/util/Logger.h"
+
+#include "DataObject.h"
+#include "TimeoffsetVariable.h"
+#include "Transforms/TransformBuilder.h"
+
+
+namespace
+{
+    namespace ConfKeys
+    {
+        const char* Timeoffset = "time_offset";
+        const char* Referencetime = "reference_time";
```

Comment:
Please camel case these variables to be consisitent with variable arguments for other variables. Should be **timeOffset**, and **referenceTime** instead.